### ISSUE TITLE
Sensor unavailable after restart

### ISSUE THREAD

**Issue by Pulpyyyy** [vingerha/gtfs2#6]
Hello,

After retarting HA, sensors are not available
```
2023-11-12 21:03:15.877 ERROR (MainThread) [custom_components.gtfs2.gtfs_helper] The given GTFS zipfile was not found
2023-11-12 21:03:15.965 ERROR (MainThread) [custom_components.gtfs2.coordinator] Unexpected error fetching 7b4b63203f5255ac347e71621891f771 data: 'str' object has no attribute 'engine'
File "/config/custom_components/gtfs2/coordinator.py", line 73, in _async_update_data
File "/config/custom_components/gtfs2/gtfs_helper.py", line 437, in check_datasource_index
2023-11-12 21:03:15.974 ERROR (MainThread) [homeassistant.components.sensor] Error while setting up gtfs2 platform for sensor
File "/config/custom_components/gtfs2/coordinator.py", line 73, in _async_update_data
File "/config/custom_components/gtfs2/gtfs_helper.py", line 437, in check_datasource_index
File "/config/custom_components/gtfs2/sensor.py", line 70, in async_setup_entry
2023-11-12 21:03:16.087 ERROR (MainThread) [homeassistant.components.sensor] Error while setting up gtfs2 platform for sensor
File "/config/custom_components/gtfs2/sensor.py", line 73, in async_setup_entry
GTFSDepartureSensor(coordinator),
File "/config/custom_components/gtfs2/sensor.py", line 98, in __init__
File "/config/custom_components/gtfs2/sensor.py", line 156, in _update_attrs
2023-11-12 21:03:16.097 ERROR (MainThread) [homeassistant.components.sensor] Error while setting up gtfs2 platform for sensor
File "/config/custom_components/gtfs2/sensor.py", line 73, in async_setup_entry
GTFSDepartureSensor(coordinator),
File "/config/custom_components/gtfs2/sensor.py", line 98, in __init__
File "/config/custom_components/gtfs2/sensor.py", line 156, in _update_attrs
```

**Comment by vingerha** [comment]
Interesting, I restart HA a lot during development and never had this, which version?

**Comment by vingerha** [comment]
Resolved in dev release 0.1.6c1
